### ISSUE TITLE
Prompt not working

### ISSUE THREAD

**Issue by Whylde-Tech** [zap-zsh/zap#93]
Hey Guys.  Just started to fool around with this.  But the prompt plugin isn't affecting my zsh prompt.  Any ideas?

**Comment by mamaraddio** [comment]
What prompt where you using before zap-prompt?

**Comment by wintermi** [comment]
We require more information before we can help resolve this issue.  A new bug reporting template now exists which guides you through the minimum detail we require to help.  We are therefore closing this issue and would respectfully request that you raise a new issue, completing the bug report should you still be experiencing the problem.
